pegasus/sites.v3/code.org/public/educate/professional-learning.md

```diff
@@ -6,39 +6,58 @@ theme: responsive
 
+<br><br>
 
-### Summer Workshop
-Summer 5 day in-person professional learning workshop, followed by online material.
+#### 7-9th grade teachers: Computer Science Discoveries
 
-- Focused on quality computer science pedagogy—Teaching content through inquiry and equity
-- Explore lessons as a teacher and a learner 
-- Creating a professional learning community that will extend beyond the Code.org partnership.
+For educators in 7-9th grade classrooms, our Computer Science Discoveries course is a great place to start. The course can be taught as a semester or full-year introduction and covers a broad range of topics such as physical computing, building websites, programming and learning about data. The program is currently full for the 2017-18 school year, but sign up on our interest form today if you'd like to apply next year.
 
-### Ongoing Support
-Quarterly in-person and/or online professional learning workshops combined with online support.
+[<button>Learn more</button>](/educate/csd)
 
-- Provide content and pedagogical support for upcoming units
+<br><br>
 
-[<button>Inquire about a regional partnership</button>](/educate/partner)
+#### 9-12th grade teachers: Computer Science Principles
 
+If you're teaching 9-12th grade students, our Computer Science Principles course can be taught as an AP or non-AP course. Either way, there are no prerequisites required. The course is designed to be far more than a traditional introduction to programming - it is a rigorous, engaging, and approachable course. It explores many of the big, foundational ideas of computing so all students understand how these concepts are transforming the world we live in. We have a few workshop seats left this summer in select regions. Or, join our interest form if you're interested in teaching in the 2018-19 school year.
 
----
-For the intellectually curious, we've also published our [professional learning philosophy](/educate/professional-development-philosophy), which we use to guide our own team's effort as well as our work with our education partners. 
+[<button>Learn more</button>](/educate/csp)
+
+## International
+
+Teachers anywhere in the world can use our free, self-paced [online workshop](https://studio.code.org/s/K5-onlinePD) to get started. (The intended audience is primary school teachers)
+
+At this point, our in-person  workshops are only available in the United States, but the [curriculum, lesson plans, tools, and support](https://studio.code.org/courses) are available at no cost worldwide. And, [join our forums](https://forum.code.org/) to connect with other teachers for support, teaching tips, and best practices.
```

Comment:
worth mentioning which ages ranges, to compare/contrast to the goal of primary schol teachers mentioned in paragraph above?  otherwise it seems like we might still be talking about primary school content.